job_bundle_output_tests/layers/expected_job_bundle/parameter_values.yaml

```diff
@@ -48,7 +48,7 @@ parameterValues:
 - name: RenderSetupIncludeLights
   value: 'true'
 - name: ArnoldErrorOnLicenseFailure
-  value: 'false'
+  value: 'true'
```

Comment:
This seems to imply we've changed the default error behavior, which I think would be considered a breaking change.

Reply:
Naw, this is just a consequence of our tests. I talked about this in the description